src/elements/content-sidebar/Sidebar.js

```diff
@@ -81,11 +83,17 @@ class Sidebar extends React.Component<Props, State> {
     }
 
     componentDidUpdate(prevProps: Props): void {
-        const { fileId, history, isLarge, location }: Props = this.props;
-        const { fileId: prevFileId, isLarge: prevIsLarge }: Props = prevProps;
+        const { defaultPath, fileId, history, isLarge, location }: Props = this.props;
+        const { defaultPath: prevDefaultPath, fileId: prevFileId, isLarge: prevIsLarge }: Props = prevProps;
         const { isDirty, isOpen }: State = this.state;
         const isForcedSet = this.isForcedSet();
 
+        // Force navigate if host application insists
```

Comment:
add this case to the componentDidUpdate UT?

Reply:
Done.